### ISSUE TITLE
sklearn.metrics.accuracy_score gt和pred写反的问题

### ISSUE THREAD

**Issue by QianLingjun** [MontaEllis/Pytorch-Medical-Classification#2]
官方：sklearn.metrics.accuracy_score(y_true, y_pred, *, normalize=True, sample_weight=None)
本文：acc = metrics.accuracy_score(predicts, gts) 
请问这里predicts,和gts，是不是写反了？

**Comment by hczyni** [comment]
> 官方：sklearn.metrics.accuracy_score(y_true, y_pred, *, normalize=True, sample_weight=None) 本文：acc = metrics.accuracy_score(predicts, gts) 请问这里predicts,和gts，是不是写反了？

我也感觉这里写反了
